src/components/Filter/Category.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { useState } from "react";
+import "./css/category.css";
+import { Price } from "./Price";
+import { Rating } from "./Rating";
+import { Sortby } from "./Sortby";
```

Comment:
You could add an `index.js` file and from that you could export all components in that folder. E.g. in this case in the Filter folder have a index file which exports category, price, rating, sortby in this way:
```jsx
export {Category} from "./Category"
export {Price} from "./Price"
...etc
```

and then import that here as 
```jsx
import {Price, Rating, Sortby} from "../Filter"
```